MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -68,4 +86,43 @@ public async Task DeleteByIdAsync(int id)
         _context.Users.Remove(user);
         await _context.SaveChangesAsync();
     }
+    
+    private async Task<bool> CheckForFreeAsync(int id, UserDtoInput userInput)
+    {
+        var user = _context.Users.FirstOrDefault(u => u.Id == id)!;
+
+        var repeatUserEmail = user.Email == userInput.Email;
+        var repeatUserPhone = user.Phone == userInput.Phone;
+
```

Comment:
if you create bool var, it`s comfortable to read var like isUserEmailExist or isUserPhoneExist

Reply:
also, i did not see Update method implementation with new feature